vaadin-testbench-shared/src/main/java/com/vaadin/testbench/TestBenchVersion.java

```diff
@@ -0,0 +1,44 @@
+package com.vaadin.testbench;
+
+import java.util.Properties;
+
+import org.openqa.selenium.BuildInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vaadin.pro.licensechecker.LicenseChecker;
+
+public class TestBenchVersion {
+
+    public static final String testbenchVersion;
```

Comment:
Could this logic be reused in `TestBenchTestCase` somehow, seems like they both do the same.